src/js/components/Menu.js

```diff
@@ -2,17 +2,18 @@
 
 import React, { Component, PropTypes } from 'react';
 import ReactDOM from 'react-dom';
-import pick from 'lodash/object/pick';
-import keys from 'lodash/object/keys';
+import classnames from 'classnames';
+import isFunction from 'lodash/lang/isFunction';
```

Comment:
We are trying to remove the lodash dependency since we are using ES6. Is it possible to test for this without lodash?